### PR TITLE
Updates on GMT, Language Moderators, Support Team and osu! Alumni Articles.

### PR DESCRIPTION
**Pull request by WalterToro**
Updates multiple articles related to People:
- Updates Global Moderator Team EN and ES articles with current roles.
- Updates Language Moderator, Support Team and osu! alumni articles and all translations

### CODE REVIEW

**Comment by Joehuu** [comment]
Don't know if I should do formatting reviews to this. There are many issues that aren't shown in this PR, so idk what to do.

I think you shouldn't mix formatting, updating the lists, and adding flags in one PR. Plus with translations, it'll be harder to review. I'll try my best though, first by reviewing what's on the PR.

**Comment by WalterToro** [comment]
Most of the translations are from previous commits so there isnt changes to words or anything else, flags were double checked and mostly copy pasted, the only possible problem might be that I might forgot someone going into alumni, but still i checked the [Promotion Log](https://osu.ppy.sh/community/forums/topics/179514) since the last commit to this file.
Yes, its still a lot of work but thats because of a lot of files being outdated for a long time (and some are still outdated)

**Comment by Joehuu** [comment]
Any reason to align the `|` on some columns? Please see the [ASC PR Tables Section](https://github.com/MegaApplePi/osu-wiki/blob/c69c3c6cadcc98a7be8f9f1944af3555c65ebcc0/wiki/Article_Styling_Criteria/en.md#tables) and apply the correct format.

**Comment by WalterToro** [comment]
Removed the beautify, even if havent been following the ASC discussion i would add that "correct" beautified tables shouldnt be that big of an issue unless those break the order on a text editor or there is an abuse on the spacing.
Also extra note, enforcing something that is still on a PR review and discussion state shouldnt be a thing.

**Comment by libneko** [comment]
If possible, could you update People/Global_Moderation_Team/ko.md also?

https://gist.github.com/libneko/50a17f5b3e64e12215683cc4bef163a3

File: wiki/People/Global_Moderation_Team/ko.md [hunk 1]
```diff
@@ -1,15 +1,12 @@
----
-outdated: true
----
 Global Moderation Team
 ==================
 
-**GMT**로 흔히 알려져 있는 **Global Moderation Team**은 주로 플레이어에 대한 중재, 관리자들과 커뮤니티에 관련된 사안에 중점을 두고 활동하는 osu!Staff에 속한 구성원입니다.
+**GMT** 로 흔히 알려져 있는 **Global Moderation Team** 은 주로 플레이어에 대한 중재, 관리자들과 커뮤니티에 관련된 사안에 중점을 두고 활동하는 osu!Staff에 속한 구성원입니다.
```
Comment (libneko) [inline wiki/People/Global_Moderation_Team/ko.md]: Whitespace between English characters and Korean characters is not necessary to be added, in this case.

File: wiki/People/Global_Moderation_Team/ko.md [hunk 2]
```diff
@@ -1,15 +1,12 @@
...
 
 GMT가 되는 방법
 ---------------
 
-**Global Moderation Team**이 되려면 상당히 어렵고 복잡한 과정을 거쳐야 합니다. 제2언어를 구사할 수 있는 사람들이 종종 GMT 구성원이 될 가능성이 있는 사람들로 지목되며, 이후에 팀 구성원으로서의 잠재력에 대해 논의 후 선출됩니다.
+**Global Moderation Team** 이 되려면 상당히 어렵고 복잡한 과정을 거쳐야 합니다. 제2언어를 구사할 수 있는 사람들이 종종 GMT 구성원이 될 가능성이 있는 사람들로 지목되며, 이후에 팀 구성원으로서의 잠재력에 대해 논의 후 선출됩니다.
```
Comment (libneko) [inline wiki/People/Global_Moderation_Team/ko.md]: It's same as line 4.

File: wiki/People/Language_Moderators/en.md [hunk 3]
```diff
@@ -1,40 +1,68 @@
...
 
-Most of the channels being present in osu! have their own moderators, which take care of anything that could invade or go against the atmosphere of the channel. While there are various QATs/GMTs taking care of these problems thanks to their power, there are various non-QAT chat OPs which moderate the channel in order to help the other moderators.
+Many of the language channels in osu! have specific moderators who oversee conversations and maintains the channels. While GMT members are mainly in charge of taking care of any issues, some QAT members are also available when needed.
 
-List of (current) language-specific moderators
+List of language-specific moderators
 ----------------------------------------------
 
 | Channel | Subforum | Operators |
 | ------- | -------- | --------- |
```
Comment (Joehuu) [inline wiki/People/Language_Moderators/en.md]: Fix delimiter (table align) format.

File: wiki/People/Language_Moderators/en.md [hunk 4]
```diff
@@ -1,40 +1,68 @@
...
-| `#arabic` | | |
-| `#balkan` | [Balkan](https://osu.ppy.sh/forum/t/83962) | [OnosakiHito](https://osu.ppy.sh/u/290128), [Stefan](https://osu.ppy.sh/u/626907) |
-| `#bulgarian` | | [Flanster](https://osu.ppy.sh/u/447818) |
-| `#cantonese` | |  [IamKwaN](https://osu.ppy.sh/u/1856463) |
-| `#chinese` | [中文](https://osu.ppy.sh/forum/25) | [Atrue](https://osu.ppy.sh/u/1758523), [IamKwaN](https://osu.ppy.sh/u/1856463), [Zero__wind](https://osu.ppy.sh/u/1822830) |
+| `#arabic`       | | |
```
Comment (Joehuu) [inline wiki/People/Language_Moderators/en.md]: Revert the beautify.

File: wiki/People/Support_Team/en.md [hunk 5]
```diff
@@ -16,18 +16,27 @@ If someone needs any assistance on the said subforums, they can be first approac
 Team Members
 ------------
 
-*Every Support Team member speaks English unless specified otherwise. Link to [user group page](https://osu.ppy.sh/g/22).*
+*Every Support Team member speaks English unless specified otherwise. Link to [user group page](https://osu.ppy.sh/groups/22).*
 
-| Name | Languages |
+| Name | Language(s) |
 | ---- | --------- |
```
Comment (Joehuu) [inline wiki/People/Support_Team/en.md]: Fix the table align format.

File: wiki/People/osu!_Alumni/en.md [hunk 6]
```diff
@@ -2,201 +2,209 @@ osu! Alumni
 ===========
 
 The **osu! Alumni** are those known for their contributions who have since moved on. Were the resources available we would erect a statue for each member in the town square.
-Link to [user group page](https://osu.ppy.sh/g/16)
+Link to [user group page](https://osu.ppy.sh/groups/16)
 
 | Name  | Epitaph |
 | ----- | ------- |
```
Comment (Joehuu) [inline wiki/People/osu!_Alumni/en.md]: Fix the table align format.

**Pull request closed without merging**